plugins/pubIds/urn/templates/settingsForm.tpl

```diff
@@ -8,165 +8,102 @@
-		<td class="value">
-			<input type="checkbox" name="checkNo" id="checkNo" value="1"{if $checkNo} checked="checked"{/if} />
-			<label for="checkNo">{translate key="plugins.pubIds.urn.manager.settings.checkNo.label"}</label><br />
-		</td>
-	</tr>
-	<tr><td colspan="2">&nbsp;</td></tr>
-	<tr>
-		<td class="label">{fieldLabel name="namespace" required="true" key="plugins.pubIds.urn.manager.settings.namespace"}</td>
-		<td class="value">
-			<select name="namespace" id="namespace" class="selectMenu">
-				<option value="">{translate key="plugins.pubIds.urn.manager.settings.namespace.choose"}</option>
-				{html_options options=$namespaces selected=$namespace}
-			</select>
-			<br />
-			<span class="instruct">{translate key="plugins.pubIds.urn.manager.settings.namespace.description"}</span>
-		</td>
-	</tr>
-	<tr><td colspan="2">&nbsp;</td></tr>
-	<tr>
-		<td class="label">{fieldLabel name="urnResolver" required="true" key="plugins.pubIds.urn.manager.settings.urnResolver"}</td>
-		<td class="value"><input type="text" name="urnResolver" value="{$urnResolver|escape}" size="40" maxlength="255" id="urnResolver" class="textField" />
-		<br />
-		<span class="instruct">{translate key="plugins.pubIds.urn.manager.settings.urnResolver.description"}</span>
-		</td>
-	</tr>
-	<tr><td colspan="2">&nbsp;</td></tr>
-	<tr>
-		<td class="label">&nbsp;</td>
-		<td class="value">
-			<span class="instruct">{translate key="plugins.pubIds.urn.manager.settings.clearURNs.description"}</span>
-			<br />
-			<input type="submit" name="clearPubIds" value="{translate key="plugins.pubIds.urn.manager.settings.clearURNs"}" onclick="return confirm({translate|json_encode key="plugins.pubIds.urn.manager.settings.clearURNs.confirm"})" class="action"/>
-		</td>
-	</tr>
-</table>
-{fbvFormArea id="doiSuffixReassignFormArea" title="plugins.pubIds.urn.manager.settings.clearURNs"}
-	{fbvFormSection}
-		<span class="instruct">{translate key="plugins.pubIds.urn.manager.settings.clearURNs.description"}</span><br/>
-		{include file="linkAction/linkAction.tpl" action=$clearPubIdsLinkAction contextId="urnSettingsForm"}
-	{/fbvFormSection}
-{/fbvFormArea}
-
-
-{fbvFormButtons submitText="common.save"}
+	{include file="common/formErrors.tpl"}
+	{fbvFormArea id="urnObjectsFormArea" title="plugins.pubIds.urn.manager.settings.urnObjects"}
+		{fbvFormSection list="true" description="plugins.pubIds.urn.manager.settings.explainURNs"}
+			{if $enableIssueURN}
+				{assign var="checked" value=true}
+			{else}
+				{assign var="checked" value=false}
+			{/if}
+			{fbvElement type="checkbox" label="plugins.pubIds.urn.manager.settings.enableIssueURN" id="enableIssueURN" maxlength="40" checked=$checked}
+			{if $enableArticleURN}
+				{assign var="checked" value=true}
+			{else}
+				{assign var="checked" value=false}
+			{/if}
+			{fbvElement type="checkbox" label="plugins.pubIds.urn.manager.settings.enableArticleURN" id="enableArticleURN" maxlength="40" checked=$checked}
```

Comment:
Just FYI, you can use...

    checked=$enableArticleURL|compare:true

(OTOH)